build-system/tasks/gen-codeowners/test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+var test = require('ava');
+var m = require('./');
+
+test('sync - build out correct CODEOWNERS', t => {
```

Comment:
Can we test that the generated file is equal to what would be generated today and fail otherwise with instructions for how to regen?